contrib/build-dokku.Dockerfile

```diff
@@ -59,4 +61,26 @@ RUN PLUGIN_MAKE_TARGET=${PLUGIN_MAKE_TARGET} \
   GOARCH=arm make version copyfiles \
   && DOKKU_ARCHITECTURE=armhf GOARCH=arm make deb-dokku
 
+
+FROM builder as arm64
+
+COPY --from=amd64 /tmp /tmp
```

Comment:
Probably should copy from `armhf` instead, as that will include the amd64 files.

Reply:
It's all the same, no?

Reply:
Ups.

Reply:
`arm64` you mean.

Reply:
Hm. Shouldn't it be
```
FROM builder as armhf

COPY --from=armhf /tmp /tmp
COPY --from=amd64 /usr/local/share/man/man1/dokku.1 /usr/local/share/man/man1/dokku.1-generated
...
FROM builder as arm64

COPY --from=arm64 /tmp /tmp
COPY --from=amd64 /usr/local/share/man/man1/dokku.1 /usr/local/share/man/man1/dokku.1-generated
```

Reply:
OK, now I get the flow. This is incrementally adding and so chaining the folders. That was not obvious.